cookbooks/aws-parallelcluster-config/templates/default/directory_service/sssd.conf.erb

```diff
@@ -20,6 +20,10 @@ ldap_referrals = False
 <%= "#{param} = #{value}" %>
   <% end %>
 <% end %>
+<% if node['cluster']['directory_service']['ldap_access_filter'] %>
+access_provider = ldap
```

Comment:
Why using `ldap` as the `access_provider`?

Reply:
It is required to enable the filter.
See references for further details on that.